### ISSUE TITLE
Error during env stopping process

### ISSUE THREAD

**Issue by leoner-omnipro** [rewardenv/reward#34]
After upgrading from v0.3.25-beta to v0.4.0 I am getting this error when I try to stop the environment I am in.

```bash
INFO    Pausing mutagen sync session...
sh: 1: mutagen: not found
ERROR   error running env command: an error occurred while configuring the command: an error occurred while pausing mutagen sync: cannot pause mutagen sync session: error running command: sh: exit status 127
```

**Screenshots**
![image](https://user-images.githubusercontent.com/93623882/217288421-bc99c106-7f13-45cf-aa82-b2973aab4f71.png)
![image](https://user-images.githubusercontent.com/93623882/217288564-88577659-2237-42d3-a071-2cbcf05c7f1e.png)

**Specifications**
- Docker desktop with WSL2
- Ubuntu 22.04
- Magento version `magento/magento-enterprise-metapackage[2.4.3]`

**Comment by janosmiko** [comment]
Hello @leoner-omnipro .

If you were using the WSL2 Direct Mount option (`REWARD_WSL2_DIRECT_MOUNT`) previously, you should configure `REWARD_SYNC_ENABLED=false` instead.

That option was removed in this release. See the breaking changes section.
https://github.com/rewardenv/reward/releases/tag/v0.4.0

**Comment by janosmiko** [comment]
I found the issue and hopefully fixed in version 0.4.1.
